maltoolbox/language/languagegraph.py

```diff
@@ -1585,29 +1582,29 @@ def _generate_graph(self) -> None:
                             )
                         )
 
+                    # TODO: attack step does not have own_requires defined in constructor
                     attack_step.own_requires = []
                     for step_expression in step_expressions:
-                        _, \
-                        result_expr_chain, \
-                        _ = \
+                        _, result_expr_chain, _ = \
                             self.process_step_expression(
                                 attack_step.asset,
                                 None,
                                 step_expression
                             )
+                        # TODO: result_expr_chain can be None
                         attack_step.own_requires.append(result_expr_chain)
```

Comment:
Not for a requirement it cannot. The only time you can get a None expression chain is if the attack step you are trying to link is local to the same asset as the originating attack step and requirements do accept take attack step names at all.